CougSat1-ADCS/src/drivers/ADCS.h

```diff
@@ -7,41 +7,41 @@
  * Unauthorized copying of this file, via any medium is strictly prohibited.  *
  ******************************************************************************/
 /**
- * @file ADCS.cpp
- * @author Cody Sigvartson
- * @date 7 October 2018
- * @brief Controls all intracommunication on the satellite
+ * @file ADCS.h
+ * @author Ryal O'Neil
+ * @date 2019-03-24
+ * @brief Initializes subsystem objects and controls priority queue
  *
  * ADCS Tasks:
  *	-Initialize subsystems
  *  -Read 4x IMUs
  *  -Read 1x GPS
 	-Reach out to hbridge, provide it a PWM duty cycle signal to tell it how much we want it to run and	which direction
 	-Read thermistor of magnetorquer
-	-Get power info from IHU
-	-Photodiodes?
+	-Get power info from CDH
+	-Photodiodes
  */
+
 #ifndef ADCS_H
 #define ADCS_H
 
 #include <mbed.h>
 #include <rtos.h>
 #include "ADCSPins.h"
-#include "drivers/IHUCOM/IHUCOM.h"
-
+#include "drivers/CDHCOM.h"
 
-class ADCS 
+class ADCS
 {
-	private:
-		DigitalIn I2CListen;
-		Thread MONITOR;
-		Thread I2CREAD;
-		char message[4];
-		IHUCOM Reader;
-	public:
-		ADCS();
-		void monitor_thread();
-		void i2c_thread();
-		void initialize();
+  private:
+	Thread MONITOR;
+	Thread CDHREAD;
+	char message[4];
+	CDHCOM CDH;
+	
+  public:
+	ADCS();
+	void monitor_thread();
+	void cdh_thread();
```

Comment:
Wrong formatting, read the SOP

Reply:
Also, don't have a member function be called main. There is only one main. Use initialize, run, and stop if applicable. If you really want this function to be called 'main', remove this class and use the actual main